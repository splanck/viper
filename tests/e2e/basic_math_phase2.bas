10 PRINT SIN(0#)
20 PRINT COS(0#)
30 PRINT POW(2#,10#)
