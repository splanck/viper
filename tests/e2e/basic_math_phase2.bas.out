0
1
1024
